scripts/build/rollup-plugin-javascript-prefixer.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable import/no-extraneous-dependencies */
+const glob = require('glob');
+const fs = require('fs');
+
+const componentPackageJson = require(`${process.cwd()}/package.json`);
+
+const jsFilesToPrefix = glob
+  .sync('**/*.js', {
+    ignore: [
+      'node_modules/**/*',
+      'dist/**/*',
+      'lib/**/*',
+      'index.react.js',
+      '**demo.js',
+      '**.bak',
+      '**test**',
+      '**story**',
+    ],
+  })
+  .map(f => `./${f}`);
+
+const _getComponentInfo = () => {
+  const elementMapping = new Map();
+  elementMapping.set('10-atoms', 'a-');
+  elementMapping.set('20-molecules', 'm-');
+  elementMapping.set('30-organisms', 'o-');
+
+  const cwdAsStringArray = process.cwd().split('/');
+  const customElementName = componentPackageJson.name.replace('@axa-ch/', '');
+  const customElementTypePrefix = elementMapping.get(
+    cwdAsStringArray[cwdAsStringArray.length - 2]
+  ); // atom (a-) / molecule (m-) / organism (o-)
+  const newPrefix = `nva${componentPackageJson.version.replace(/\./g, '-')}`;
+  return {
+    prefix: newPrefix, // 'nva1-2-3'
+    standardComponentClassPrefix: customElementTypePrefix + customElementName, // a-button-link
+    cssPrefix: `${newPrefix}_${customElementTypePrefix}${customElementName}`, // .nva1-2-3_a-button-link
+  };
+};
+
+export const componentInfo = _getComponentInfo();
```

Comment:
While I understand this wish, I'm not sure about this. If I move it down, I have to declare the const and only at the bottom export it. When you export it where you declare it, it adds to readability. Otherwise you see an export that you need to jump into first to get the context.

Reply:
Ich bin da wohl etwas schnell drüber gegangen. Jetzt sehe ich, dass die function genau drüber steht. Ja stimmt so macht das Sinn. Aber noch ein anderer Vorschlag dazu:

Du machst eine Methode öffentlich, die `_.....`heisst, also intern sein soll. Warum nicht `_`weglassen und direkt exporten? Also so:
`export const getComponentInfo`

Reply:
Because the method would then execute all the time, instead of executing once and then always return that result (result is stored in the value).

Reply:
@raphaellueckl why do you make first a private function and then export it? doesn't make sense to me. Please export it directly

Why is it called more then once?

Reply:
@LucaMele This is rollup behavior. We run multiple configs. I can show you p2p.